dev/kyuubi-extension-spark-3-1/src/test/scala/org/apache/spark/sql/KyuubiExtensionSuite.scala

```diff
@@ -62,6 +59,8 @@ class KyuubiExtensionSuite extends QueryTest with SQLTestUtils with AdaptiveSpar
     if (_spark != null) {
       _spark.stop()
     }
+    Utils.deleteRecursively(new java.io.File("spark-warehouse"))
+    Utils.deleteRecursively(new java.io.File("metastore_db"))
```

Comment:
I see your requirements. We can share the one spark session between all test in extension, so we don't need do this one by one.

Reply:
What can we do if we want to share a single session between two `Suites`?

Reply:
I think I see.

Reply:
Use a singleton class to init spark session ?